### PR TITLE
debian: Add ibacm-dev package

### PR DESCRIPTION
**Pull request by rfehren**
Hi Benjamin,

could you please apply this patch, build and upload the new deb to unstable? The new package opa-ff (currently in the NEW queue) needs ibacm-dev a as a build dependency. It would also be nice if you could put the package under the debian-hpc salsa group umbrella.

Thanks,

Roland

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -42,6 +42,17 @@ Description: RDMA core userspace infrastructure and documentation
...
+Package: ibacm-dev
+Section: libdevel
+Architecture: linux-any
+Multi-Arch: same
+Depends: ${misc:Depends}
+Breaks: ibacm (<< 18.1-1)
+Replaces: ibacm (<< 18.1-1)
+Description: Development files for the ibacm library
+ This package is needed to compile ibacm providers. It contains just the ibacm
+ header files. The corresponding library is part of the ibacm package.
```
Comment (bdrung) [inline debian/control]: For consistency, please copy the description of ibacm to this package.

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
+rdma-core (18.1-2) unstable; urgency=medium
+
+  * Add ibacm-dev package
+
+ -- Roland Fehrenbacher <rf@q-leap.de>  Fri, 15 Jun 2018 08:36:17 +0000
```
Comment (bdrung) [inline debian/changelog]: The release should be put into a separate commit.

File: debian/ibacm-dev.install [hunk 3]
```diff
@@ -0,0 +1,2 @@
+usr/include/infiniband/acm.h
+usr/include/infiniband/acm_prov.h
```
Comment (bdrung) [inline debian/ibacm-dev.install]: Also `ibacm.7` and `ibacm_prov.7` should be placed in this -dev package.

File: debian/ibacm.install [hunk 4]
```diff
@@ -2,8 +2,6 @@ etc/init.d/ibacm
 lib/systemd/system/ibacm.service
 lib/systemd/system/ibacm.socket
 usr/bin/ib_acme
-usr/include/infiniband/acm.h
-usr/include/infiniband/acm_prov.h
 usr/lib/*/ibacm/libibacmp.so
```
Comment (bdrung) [inline debian/ibacm.install]: probably `libibacmp.so` should be placed in the dev package too.

**Comment by nmorey** [comment]
@bdrung Please let me know if this needs to be backported to stable branches

**Comment by rfehren** [comment]
I've created a separate PR for the changes to master.

> probably libibacmp.so should be placed in the dev package too.

No, this should stay in ibacm, it is supposed to be dlopened  and not compiled against.
